File: solutions/problem_0093/java/Calculadora.java
public class Calculadora {

    public int soma(int arg1, int arg2) {
        return arg1 + arg2;
    }

    public int multiplicacao(int arg1, int arg2) {
        return sucessiveSum(arg1, arg2);
    }

    private int sucessiveSum(int base, int multiplier) {
        int result = base;
        for (int i = 1; i != multiplier; i++) {
            result += base;
        }
        return result;

    }

    public static void main(String[] args) {
        Calculadora calc = new Calculadora();
        int a = 5;
        int b = 8;

        System.out.println(a + " + " + b + " = " + calc.soma(a, b) );
        System.out.println(a + " x " + b + " = " + calc.multiplicacao(a, b) );
    }
}

File: solutions/problem_0093/kotlin/Calculadora.kt
class Calculadora {

    fun soma(arg1: Int, arg2: Int) = arg1 + arg2

    fun multiplicacao(arg1: Int, arg2: Int) = sucessiveSum(arg1, arg2)

    private fun sucessiveSum(base: Int, multiplier: Int): Int{
        var result = 0
        for (i in (1..multiplier)){
            result += base
        }
        return result
    }
}

fun main() {
    val calc = Calculadora()
    val a = 5
    val b = 8

    println("$a + $b = ${calc.soma(a, b)}")
    println("$a x $b = ${calc.multiplicacao(a, b)}")
}